doc/sphinx/language/core/index.rst

```diff
@@ -30,10 +30,67 @@ will have to check their output.
 .. toctree::
    :maxdepth: 1
 
+   .. The two chapters Gallina specification language and CIC are
+      planned for removal.  Eventually, the present "Core language"
+      chapter will play the role of the former Gallina chapter, except
+      that some core notions will have moved there from the former
+      Gallina extensions chapter.
+
    ../gallina-specification-language
    ../cic
+
+   .. Comments are used to show upcoming sections (not yet split out).
+
+   .. Lexical conventions, how to read the grammars in the manual,
+      presentations of the basic notions: terms, commands, tactics,
+      attributes and options.
+
+   .. Terms, typing and conversion (only basic constructs:
+      abstraction, product, application, type cast, brief mention of
+      sorts).
+
+   .. Assumptions
```

Comment:
What goes under "Assumptions"?

Reply:
The current "Assumptions" section of the Gallina chapter.